src/actions/notes.js

```diff
@@ -59,8 +60,8 @@ const postUpdateNote = (bookmarkId, { noteId, newText, text }) => {
         { type: UPDATE_NOTE_FAILURE, meta },
       ],
       method: 'POST',
-      body: JSON.stringify({ text: newText }),
-      endpoint: `${API_VERSION}/notes/${noteId}`,
+      body: JSON.stringify(newNoteFields),
+      endpoint: `${API_VERSION}/notes/${newNoteFields.id}`,
```

Comment:
`id` should be taken from `oldNote`, not new one (it is impossible to change not id). Also, `newNoteFields` should not include `id` (as server does not accept it in this call)